intelmq/tests/bots/experts/cymru_whois/test_expert.py

```diff
@@ -68,7 +68,6 @@
 
 @test.skip_redis()
 @test.skip_internet()
-@test.skip_travis()
```

Comment:
These tests failed regularly in the past and to not cause confusion for pull requestors for failings tests, we disabled the flaky tests in the past.

Reply:
Fixed